Project.toml

```diff
@@ -2,18 +2,24 @@ name = "Attractors"
 uuid = "f3fd9213-ca85-4dba-9dfd-7fc91308fec7"
 authors = ["George Datseris <datseris.george@gmail.com>", "Kalel Rossi", "Alexandre Wagemakers"]
 repo = "https://github.com/JuliaDynamics/Attractors.jl.git"
-version = "1.7.1"
+version = "1.6.1"
 
 [deps]
+BlackBoxOptim = "a134a8b2-14d6-55f6-9291-3336d3ab0209"
+CairoMakie = "13f3f980-e62b-5c42-98c6-ff1f3baf88f0"
```

Comment:
All these packages like Cairo and FFMPEG have no reason to be in the dependencies. Also ordinary diff eq and predefined systems. We don't use predefined systems in the test suties, we define everything from scratch